docker/Dockerfile_Process

```diff
@@ -92,4 +95,23 @@ RUN if [ "true" = "$InstallMediaPush" ]; then \
 #       Example usage: ./start.sh -e 60
 
 
+##################### supervisor configuration ##############################
+#RUN apt-get update && apt-get install -y supervisor
```

Comment:
Please run these options with a parameter. You can use --build-arg or runtime parameters which one fits better. 

Please add some test deployment scripts to make sure that it's working as expected